docs/cisco.ios.ios_ntp_module.rst

```diff
@@ -14,6 +14,13 @@ Version added: 1.0.0
    :local:
    :depth: 1
 
+DEPRECATED
+----------
+:Removed in collection release after 
+:Why: 
```

Comment:
Did this not get populated?

Reply:
Yes, This is breaking Sanity I am wondering why.